pyGHDL/libghdl/errorout.py

```diff
@@ -1,3 +1,8 @@
+# Auto generated Python source file from Ada sources
+# Use xtools via make to regenerate:
```

Comment:
`xtools` does not exist now. Scripts are `scripts/pnodes.py` and `scripts/pnodespy.py`. I suggest rewording to: `Use scripts/pnodes*.py via make to regenerate:`

Reply:
After squashing my commits the headers didn't get update or other wired things happend in Git ...

That's what the script writes out, the header you referenced is an old version. I'll push an updated version.
```
# Auto generated Python source file from Ada sources
# Call 'make' in 'src/vhdl' to regenerate:
```

Reply:
@umarcor I frorce-pushed (2ad051e). Please check again.

Reply:
LGTM!